ibacm/src/acm.c

```diff
@@ -3228,6 +3234,7 @@ static void acm_log_options(void)
 	};
 
 	acm_log(0, "log file %s\n", log_file);
+	acm_log(0, "umad debug level %d\n", umad_debug_level);
```

Comment:
My previous review is missing a comment I had about this change. Can you not place his between log_file and log_level since they are related?